systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load url from future %}
+{% load i18n %}
```

Comment:
Even thought this is good practice, we are not doing any translations. So you safely remove this and all `trans` tags.